xml/bura-manual_services.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0"?>
+<!DOCTYPE section [
+ <!ENTITY % entities SYSTEM "entity-decl.ent"> %entities;
+]>
+<section xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude"
+        xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="topic_bxm_gxr_st">
+ <title>Manual Backup Overview</title>
+ <para>
+  This section covers manual backup and some restore processes. Full
+  documentation for restore operations is in <xref
+  linkend="unplanned_maintenance"/>.To back up somewhere outside the cluster,
+  please refer to <xref linkend="topic_jsc_qps_qt"/>. The following backups are
+  covered:
+ </para>
+ <itemizedlist>
+  <listitem>
+   <formalpara>
+    <title>&clm; Data</title>
+    <para>
+     All important information on the &clm;
+    </para>
+   </formalpara>
+  </listitem>
+  <listitem>
+   <formalpara>
+    <title>&mariadb; database that is part of the Control Plane</title>
+    <para>
+     The &mariadb; database contains most of the data needed to restore
+     services. &mariadb; supports full back up and recovery for all services.
+     Logging data in &elasticsearch; is not backed up. &swift; objects are
+     not backed up because of the redundant nature of &swift;.
+    </para>
+   </formalpara>
+  </listitem>
+  <listitem>
+   <formalpara>
+    <title>&swift; Rings used in the &swift; storage deployment </title>
+    <para>
+     &swift; rings are backed up so that you can recover more quickly even
+     though &swift; can rebuild the rings without this data. Automatically
+     rebuilding the rings is slower than restoring via a backup.
+    </para>
+   </formalpara>
+  </listitem>
+  <listitem>
+   <formalpara>
+    <title>Audit Logs</title>
+    <para>
+     Audit Logs are backed up to provide after-the-fact information and
+     statistical data for performance and security purposes.
+    </para>
+   </formalpara>
+  </listitem>
+ </itemizedlist>
+ <para>
+  The following services will be backed up; the data needed to restore the
+  services is backed up. This includes databases and configuration-related
+  files.
+ </para>
+ <important>
+  <para>
+   However, data content for some services is not backed up, as indicated below.
```

Comment:
Remove "however" "Data content is not backed up for all services as indicated below."